lib/reporting/proofing_rate_report.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'reporting/identity_verification_report'
+
+module Reporting
+  class ProofingRateReport
+    DATE_INTERVALS = [30, 60, 90]
+
+    # The basic goal here is to call IdentityVerificationReport for 30, 60, and 90 days
+    # And then return some of the stuff.
+
+    attr_accessor :ivr
+
+    def report_for(start_date:)
+      csv = []
+      DATE_INTERVALS.each do |interval|
+        @ivr = Reporting::IdentityVerificationReport.new(
+          issuers: nil, # all issuers
+          time_range: (start_date - interval.days)..start_date,
+        )
+
+        csv << [interval, '-DAY RANGE PROOFING RATES']
+        csv << ['Blanket Proofing Rate ', blanket_proofing_rate]
+        csv << ['Intent Proofing Rate ', intent_proofing_rate]
+        csv << ['Actual Proofing Rate ', actual_proofing_rate]
```

Comment:
ok so I guess we didn't talk about what we wanted the final CSV to look like, but in my mind, 30/60/90 would be columns:

| | trailing 30d | trailing 60d | trailing 90d |
| --- | ---- | ---- | ---- |
| Blanket Proofing Rate | x% | y% | z% |
| Intent Proofing Rate | x% | y% | z% |
| Actual Proofing Rate | x% | y% | z% |

Also, I think it's a bad idea to reassign `@ivr` each time here, I'd consider making it a local variable and passing it in to the various helpers, so like

```ruby
report = Reporting::IdentityVerificationReport.new(....)
blanket_proofing_rate(report)
```